backend/init.sql

```diff
@@ -49,17 +50,20 @@ VALUES ('12345', 'review_id1', 'review_id2');
 CREATE TABLE review (
     review_id SERIAL PRIMARY KEY,
     movie_id VARCHAR(255) NOT NULL,
+    movie_name VARCHAR(255) NOT NULL,
     user_id VARCHAR(255) DEFAULT 'Deleted User',
+    user_email VARCHAR(255) DEFAULT 'Deleted User',
     description TEXT,
     rating INT,
     timestamp VARCHAR(255) DEFAULT CURRENT_TIMESTAMP,
     CONSTRAINT fk_user_id FOREIGN KEY (user_id) REFERENCES users (user_id) ON DELETE SET DEFAULT,
+    CONSTRAINT fk_user_email FOREIGN KEY (user_email) REFERENCES users (email) ON DELETE SET DEFAULT,
     CONSTRAINT unique_review UNIQUE (movie_id, user_id)
 );
 
 -- Insert dummy data into review table
-INSERT INTO review (movie_id, user_id, description, rating) 
-VALUES ('12345', 'auth0_sub64654', 'good movie', 4);
+INSERT INTO review (movie_id, movie_name, user_id, user_name, description, rating) 
```

Comment:
user_name 

should be user_email

![image](https://github.com/user-attachments/assets/709ef40d-7da6-449c-a588-4693a8978987)